app/models/BannerDesign.scala

```diff
@@ -9,6 +27,7 @@ case class BannerDesignImage(
 
 case class BannerDesign(
     name: String,
+    status: BannerDesignStatus,
```

Comment:
I've kept this status type separate from the status which channel tests have as banner designs are not yet goign to be archivable.